vip-application/src/main/java/fr/insalyon/creatis/vip/application/client/bean/boutiquesTools/BoutiquesApplication.java

```diff
@@ -33,6 +33,8 @@ public class BoutiquesApplication implements IsSerializable {
     private Set<BoutiquesOutputFile> outputFiles = new HashSet<>();
     private Map<String, String> tags = new HashMap<>();
     private String jsonFile;
+    private static Set<String> commandLineFlag;
+    private Set<String> customProperties;
```

Comment:
We should not store the commandLineFlag inputs in a separate field, the information is already stored in the inputs and that should be a duplicate. You should filter the flag inputs from all the inputs when you need them.

Reply:
filter information from inputs to get the commandLineFlag inputs instead of storing in a seperate field